questionnaire-en.json

```diff
@@ -344,8 +335,8 @@
         },
         {
           "correctAnswer": "Yes",
-          "question": "Are the copies of the Compliance Artifacts archived for at least as long as the Supplied Software is offered or as required by the Identified Licenses (whichever is longer)?",
-          "number": "4.d",
+          "question": "Are the Compliance Artifacts archived at least as long as the Supplied Software is offered and as required by the Identified Licenses?",
+          "number": "4.c",

```

Comment:
We should keep the numbers consistent so for compatibility with the previous responses.

Reply:
This was an error in numbering in the questionnaire itself. From your note I understand that if we correct the error for the OpenChain Specification 2.0 self-certification questionnaire it would break the pre-existing database of conformant or partially conformant organizations. Is that correct?

Reply:
@shanecoughlan That is correct, fixing the numbering issue would require a major version change.  With the number of changes, a major version change which allows for breaking compatibility may be in order.